### PR TITLE
Slime grinders now suck up more than one slime

### PR DESCRIPTION
**Pull request by coiax**
Fixes issue #16894

:cl: coiax
bugfix: Fixed slime grinders not sucking up more than one slime
/:cl:

### CODE REVIEW

**Comment by Core0verload** [comment]
Nice one-liner.

**Comment by LatD** [comment]
What does that if-check even do now? What's the point of it?

**Comment by Core0verload** [comment]
It stops the machine from sucking in slimes that are already in.

File: code/modules/food&drinks/kitchen machinery/processor.dm [hunk 1]
```diff
@@ -34,7 +34,7 @@
 	var/mob/living/simple_animal/slime/picked_slime
 	for(var/mob/living/simple_animal/slime/slime in range(1,src))
 		if(slime.loc == src)
```
Comment (ChangelingRain) [inline code/modules/food&drinks/kitchen machinery/processor.dm]: This shouldn't even DO anything, range() shouldn't get a slime that's literally inside the machine already.

**Comment by coiax** [comment]
Well, Lati's bug isn't possible if it's picking up slimes already in the machine.

@Core0verload Also, I thought that's what the tweak image in the changelog was for. I can take it out if you think that's better?

**Comment by KorPhaeron** [comment]
I think there is a bugfix tag. Doesnt really matter either way, it's fine if you want to changelog a bugfix

**Comment by KorPhaeron** [comment]
Sort of related: I wonder if changelogging every bugfix would make people appreciate that coders do more than just randomly ruin the game, or just make people skip the changelog.

**Comment by coiax** [comment]
Make a changelog that includes bugfixes? Or just put them at the end of the day with no author attribution in small font?

**Comment by KorPhaeron** [comment]
@MrStonedOne

How hard would it be to give bugfixes a separate tab in changelogs?

**Pull request merged**